src/content/configuration/resolve.md

```diff
@@ -115,6 +115,31 @@ The following table explains other cases:
 
 W> `resolve.alias` takes precedence over other module resolutions.
 
+W> [string] and `false` values are supported since webpack 5.0.x.
+
+```js
+module.exports = {
+  //...
+  resolve: {
+    alias: {
+      _: [path.resolve(__dirname, 'src/utilities/'), path.resolve(__dirname, 'src/templates/')]
+    }
+  }
+};
+```
+
+`resolve.alias` when set to `false` will ignore a module.
```

Comment:
I am not really clear from reading it, what exactly happens, webpack won't bundle it? e.g. import will not result in an error and its value will be `undefined`? e.g.
```
import x from './ignored-module';

console.log(x); // undefined
```

Or no?

Also text can be improved:
```suggestion
Setting `resolve.alias` to `false` will tell webpack to ignore a module.
```